nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -288,10 +288,14 @@ def _set_const_value(node_with_const: ov.Node,
                          const_value: np.ndarray) -> None:
         const_port = node_with_const.input(const_port_id)
         const_node = node_with_const.input_value(const_port_id).get_node()
-        if const_node.get_type_name() == 'Convert':
+
+        while const_node.get_type_name() != 'Constant':
```

Comment:
Is it possible that this loop leads to hang or a runtime error?

Reply:
fixed